packages/ott-vis-panel/src/colors.ts

```diff
@@ -1,12 +1,23 @@
+import * as d3 from "d3";
+
 class ColorProvider {
+	private color: d3.ScaleOrdinal<string, string>;
+	private assignments: Map<string, string>;
```

Comment:
nit: with this, it can be left out of the constructor

```suggestion
	private assignments: Map<string, string> = new Map();
```